internal/bminventory/inventory.go

```diff
@@ -1918,30 +1918,16 @@ func (b *bareMetalInventory) updateClusterData(ctx context.Context, cluster *com
 		serviceCidr = *params.ClusterUpdateParams.ServiceNetworkCidr
 		updates["service_network_cidr"] = serviceCidr
 	}
-	if params.ClusterUpdateParams.AdditionalNtpSource != nil {
-		ntpSource := swag.StringValue(params.ClusterUpdateParams.AdditionalNtpSource)
-
-		if ntpSource != "" && !validations.ValidateAdditionalNTPSource(ntpSource) {
-			err = errors.Errorf("Invalid NTP source: %s", ntpSource)
-			log.WithError(err)
-			return common.NewApiError(http.StatusBadRequest, err)
-		}
-
-		updates["additional_ntp_source"] = ntpSource
-	}
 
 	if params.ClusterUpdateParams.UserManagedNetworking != nil && swag.BoolValue(params.ClusterUpdateParams.UserManagedNetworking) != userManagedNetworking {
 		userManagedNetworking = swag.BoolValue(params.ClusterUpdateParams.UserManagedNetworking)
 		updates["user_managed_networking"] = userManagedNetworking
 		if userManagedNetworking {
-			err = validateUserManagedNetworkConflicts(params.ClusterUpdateParams, log)
+			err = setCommonUserNetworkManagedParams(params.ClusterUpdateParams, cluster, updates, log)
 			if err != nil {
 				return err
 			}
-			updates["vip_dhcp_allocation"] = false
 			vipDhcpAllocation = false
-			updates["api_vip"] = ""
-			updates["ingress_vip"] = ""
 			setMachineNetworkCIDRForUpdate(updates, "")
```

Comment:
You can move  `setMachineNetworkCIDRForUpdate(updates, "")` into
`setCommonUserNetworkManagedParams` and place it under if common.IsSingleNodeCluster(cluster)...